### ISSUE TITLE
[bug] Permission un-organized my google photos

### ISSUE THREAD

**Issue by Autumn2308** [mozilla-mobile/fenix#21137]
I allowed Firefox permission to my photos and media to download pictures from an icloud link sent to me (I have android/google photos) and immediately all of the last 3.5 months of my camera folder and all of both my messages and screenshots folders were completely unorganized. The 'details' under each photo are wrong date and wrong location. There are 'burst shots' of pictures taken in different months (and different folders). Please help. I am an avid user of my photos for projects. Major inconvenience and slightly alarming that this would happen from a permission from Firefox. :(

**Comment by kbrosnan** [comment]
This is not possible. Firefox does not have access to the files directly.

**Comment by Autumn2308** [comment]
I believe it needs a bit more consideration than that, sir. I pressed 'download' within the icloud link and it said something as 'firefox needs permission to media" , I hesitated but did so. I know for 100% it said Firefox because after going to my photos and finding the fucking up of them I turned off the permission through my phone settings. It was not allowed before. I know this as fact.On Sep 4, 2021 12:34 AM, Kevin Brosnan ***@***.***> wrote:
This is not possible. Firefox does not have access to the files directly.
